docs/hugo/content/guide/crd-management.md

```diff
@@ -0,0 +1,121 @@
+2. Install the operator Helm chart. Specify `installCRDs: false` as part of `values.yaml` when installing.
+3. Ensure the pod launches correctly.
+
+The process for upgrading the operator is similar:
+1. Upgrade the CRDs you are using. These must be the CRDs for the same version of the operator you upgrade to in step 2.
+2. Upgrade the operator Helm chart. Specify `installCRDs: false` as part of `values.yaml` when upgrading.
+3. Ensure the pod launches correctly.
+
+## With YAML
+
+### Automatic CRD installation (recommended)
+
+Unlike Helm there are no parameters built into the raw YAML. You must use a tool such as `kustomize` or `sed` to
+set the `--crd-pattern` command-line argument of the operator pod to include the CRDs you wish to install.
+
+Example: `cat operator.yaml | sed 's_--crd-pattern=.*_--crd-pattern=resources.azure.com/*;containerservice.azure.com/*_g' | kubectl apply -`
+
+Alternatively you can use `kustomize` with a configuration like:
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - https://github.com/Azure/azure-service-operator/releases/download/v2.1.0/azureserviceoperator_v2.1.0.yaml
+
+patches:
+  - patch: |-
+      - op: add
+        path: /spec/template/spec/containers/0/args/-
+        value: --crd-pattern=resources.azure.com/*;containerservice.azure.com/*;keyvault.azure.com/*;managedidentity.azure.com/*;eventhub.azure.com/*
+    target:
+      kind: Deployment
+```
+
+### Manual CRD installation
+
+If you do not want to grant the operator permission to create + update CRDs, you can instead manage the CRDs yourself.
+To do this you can use the customresourcedefinitions YAML that is part of each release. This YAML contains just the ASO CRDs.
+You can then filter to whichever subset of CRDs you want or apply them all.
+See for example [azureserviceoperator_customresourcedefinitions_v2.0.0.yaml](https://github.com/Azure/azure-service-operator/releases/download/v2.0.0/azureserviceoperator_customresourcedefinitions_v2.0.0.yaml)
+
+The process to do this for a fresh install is:
+1. Install the CRDs you want to use. These must be the CRDs for the same version of the operator you install in step 2.
+2. Install the operator YAML.
+3. Ensure the pod launches correctly.
+
+The process for upgrading the operator is similar:
+1. Upgrade the CRDs you are using. These must be the CRDs for the same version of the operator you upgrade to in step 2.
+2. Upgrade the operator YAML.
+3. Ensure the pod launches correctly.
+
+## Uninstalling CRDs
+
+ASO CRD automation will **never** under any circumstances uninstall CRDs. In fact, it doesn't have delete CRD permissions.
+This is a safety feature because removal of CRDs can be dangerous.
+
+The recommended workflow for removing CRDs you are no longer using is:
+
+1. Delete or otherwise remove all instances of the CRD in question. For example if you want to remove the 
+   `keyvault.azure.com/vault` CRD, first delete all instances of that CRD.
```

Comment:
```suggestion
0. Ensure ASO is stopped
1. Delete or otherwise remove all instances of the custom resource (CR) in question. For example, if you want to remove the 
   `keyvault.azure.com/vault` CRD, first delete all `vault` instances.
   *  If you want to retain the Vault resources in Azure, apply a `skip-reconcile` annotation first.
```

Reply:
ASO doesn't need to be stopped for step 1, because we want deletion to actually propagate to Azure (if that's what the user wants at least).

Reply:
I'll update this to clarify when you should stop (after step 2).